jellyfin-rpc/src/lib.rs

```diff
@@ -457,69 +478,82 @@ impl Client {
-                                    state +=
-                                        &format!(" {} ", self.movies_display_options.separator);
-                                }
-
-                                state += &format!("⭐{:.1}/10", community_score);
-                            }
-                        }
-                        _ => (),
-                    }
+                let year = session
+                    .now_playing_item
+                    .production_year
+                    .map(|x| x.to_string())
+                    .unwrap_or(String::new());
+
+                let critic_rating = session
+                    .now_playing_item
+                    .critic_rating
+                    .map(|x| x.to_string())
+                    .unwrap_or(String::new());
+
+                let community_rating = session
+                    .now_playing_item
+                    .community_rating
+                    .map(|x| x.to_string())
+                    .unwrap_or(String::new());
+
+                let image_text = self
+                    .music_display_options
+                    .image_text
+                    .replace("{genres}", &genres)
+                    .replace("{year}", &year)
+                    .replace("{critic_rating}", &critic_rating)
+                    .replace("{community_rating}", &community_rating);
+
+                let details_line_1 = self
+                    .music_display_options
+                    .state_text
+                    .replace("{genres}", &genres)
+                    .replace("{year}", &year)
+                    .replace("{critic_rating}", &critic_rating)
+                    .replace("{community_rating}", &community_rating);
+
+                let details_line_2 = self
+                    .music_display_options
+                    .details_text
+                    .replace("{genres}", &genres)
+                    .replace("{year}", &year)
+                    .replace("{critic_rating}", &critic_rating)
+                    .replace("{community_rating}", &community_rating);
+
+                DisplayText {
+                    image_text,
+                    state_text: details_line_1,
+                    details_text: details_line_2,
                 }
+            }
+            _ => {
+                let state = session
+                    .now_playing_item
+                    .genres
+                    .as_ref()
+                    .unwrap_or(&vec!["".to_string()])
+                    .join(", ");
 
-                state
+                DisplayText {
+                    image_text: String::new(),
+                    state_text: state,
+                    details_text: String::new(),
+                }
             }
-            _ => session
-                .now_playing_item
-                .genres
-                .as_ref()
-                .unwrap_or(&vec!["".to_string()])
-                .join(", "),
         }
     }
```

Comment:
This should be a new function instead of breaking the old one, you could use a simple match statement based on the type of media being played.

Also image text should stay the same as before on types that dont benefit from it like Movie, LiveTv, Book and Episode. Only AudioBook and Music need that line.

The default can also be the old large_image_text with the version number